### DISCUSSION TITLE
IOS Folder Marker Missing

### DISCUSSION THREAD

**Discussion by dar3d3viilx3** [MobiusSync/MobiusSync#225] [Q&A]
![image](https://github.com/user-attachments/assets/a6bb3b30-78b1-4470-9742-6dc0de7b04b4)

I bought the Pro App, and after some files synced it stopped working.

How can i fix that error?

I guess maybe, becaus on apple folder names starting with . Are restricted for the system.

Anyone can help me here how to fix or workaround that?

**Comment by rasa** [comment]
You might try installing an app that allows you to view and create "hidden files", such as the `.stfolder` marker directory. Two that might work are Documents by Readdle and FileApp. Please let us know if that works for you.

**Comment by dar3d3viilx3** [comment]
I reconnected the folders new, and now it working properly.

Maybe the issue occured due to the fact , that i had on the first failed attempt some filters on 
(.vs .thumbnails .jpg)

Anyway thanks for your reply.
